clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -60,14 +60,14 @@ const (
 	// countQuery %v should be replaced with (1) filters and (2) query window size.
 	countQuery = "sum(increase(apiserver_request_duration_seconds_count{%v}[%v])) by (resource, subresource, scope, verb)"
 
-	countSlowQuery = "sum(rate(apiserver_request_duration_seconds_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
+	countFastQuery = "sum(increase(apiserver_request_duration_seconds_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
 
 	// exclude all buckets of 1s and shorter
-	filterGetAndMutating = `verb!~"WATCH|WATCHLIST|PROXY|CONNECT", le!~"0.\\d+|1"`
+	filterGetAndMutating = `verb!~"WATCH|WATCHLIST|PROXY|CONNECT|LIST", le="1"`
```

Comment:
does le="1" work ok? Will it count all buckets?

Reply:
le="1" counts only this single bucket, which contain all request that did fit in 1s

So idea is to treat this bucket as fast queries, and subtract it from overall count to get slow queries.

Reply:
Makes sense - thanks.

So I'm actually wondering what the previous query was doing - it was choosing one of the buckets?

Anyway - thanks a lot for attacking this problem - this approach looks reasonable, let me know when this will be ready for final review.

Reply:
To be honest - I do not understand the previous approach.
But results were somewhat close (less than 2x away) to the new approach.

Reply:
Actually, old logic frequently show 0 SlowQueries even when the measurement fails. It was not noticed before as SlowQueries was always set to 0 if slow queries were not used.